Makefile

```diff
@@ -118,11 +118,11 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: generate fmt vet ## Run tests.
-	$(GO_CMD) test $(GO_TEST_FLAGS) ./pkg/... -coverprofile cover.out
+	$(GO_CMD) test $(GO_TEST_FLAGS) $(shell go list ./... | grep -v -e test -e apis) -coverprofile cover.out
```

Comment:
Previously, `main_test.go` could not be tested.

Reply:
Good catch.

Reply:
would `-e /test` work?

Reply:
Yes, It works fine.
Do you prefer to use `go list ./... | grep -v -e '/test' -e '/apis/kueue/v1alpha2'`?